PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -5,8 +5,11 @@ import Foundation
 /// All getter and setter are sideeffect free and automatically syncronize
 /// after writing.
 internal struct PiwikUserDefaults {
-    static var standard = PiwikUserDefaults()
-    let userDefaults = UserDefaults.standard
+    let userDefaults: UserDefaults
+
+    init(suiteName: String?) {
```

Comment:
> Maybe non optional?
We cannot really make it non optional without loosing compatibility with current installations. They are already using it with suiteName nil